### ISSUE TITLE
Continent strange data

### ISSUE THREAD

**Issue by MartinMikita** [OSMNames/OSMNames#135]
Looking for the continent, like `Europe` or `Asia` returns very strange JSON data:

```
    {
      "wikipedia": "en:Europe",
      "rank": 22752522.0,
      "county": "Werra-Meißner-Kreis",
      "street": "\"\"",
      "wikidata": "Q46",
      "country_code": "de",
      "osm_id": "25871341",
      "housenumbers": "\"\"",
      "id": 20,
      "city": "Sontra",
      "display_name": "Europe, Sontra, Werra-Meißner-Kreis, Regierungsbezirk Kassel, Hessen, Deutschland",
      "lon": 10.0,
      "state": "Hessen",
      "boundingbox": [
        10.0,
        51.0,
        10.0,
        51.0
      ],
      "type": "continent",
      "importance": 0.857101,
      "lat": 51.0,
      "class": "place",
      "name": "Europe",
      "country": "Germany",
      "name_suffix": "Sontra, DE",
      "osm_type": "node",
      "place_rank": 2.0,
      "alternative_names": "Avrupa,Châu Âu,Eoraip,Eropah,Eurooppa,Europa,Európa,Eŭropo,Evropa,Evrópa,Ewropa,Jeropa,Ūropi,Yurop,اروپا,أوروبا,Ευρώπη,Европа,Еўропа,Європа,यूरोप,ಯುರೋಪ್,ஐரோப்பா,歐洲(欧洲)"
    },
```

The [OSM node](http://www.openstreetmap.org/node/25871341) of the Europe.

Is this intended behaviour, or should it be fixed?

**Comment by philippks** [comment]
This is definitely a bug. To be more precise, the logic [here](https://github.com/OSMNames/OSMNames/blob/master/osmnames/prepare_data/create_hierarchy/set_parent_ids.sql#L8) where the hierarchy is created, makes no sense... I'll assign the issue to myself.

**Comment by philippks** [comment]
@MartinMikita the bug should now be solved with the release v2.0.2.
